### ISSUE TITLE
descriptions in alembic revision files gets broken

### ISSUE THREAD

**Issue by finswimmer** [PyCQA/docformatter#242]
Hey,

`alembic` adds something like this to each revision file:

```py
"""Add some column.

Revision ID: <some id>>
Revises: <some other id>
Create Date: 2023-01-06 10: 13:28.156709
"""
``` 

Starting from docformatter 1.7.1 on, `docformatter` breaks this description:

```py
"""Add some column.

Revision ID: <some id>> Revises: <some other id> Create Date
: 2023-01-06 10: 13:28.156709
"""
```

I guess this has something to do with https://github.com/PyCQA/docformatter/pull/201

Any chance to get this fixed?

fin swimmer

BTW: Thanks a lot for this wonderful tool!

**Comment by weibullguy** [comment]
@finswimmer this is fixed in v1.7.3-rc3 if you'd like to give it a try.  Or you can wait until v1.7.3 is released.

**Comment by finswimmer** [comment]
Thanks for your quick answer @weibullguy :pray: 

I tried 1.7.3-rc3. Unfortunately the behavior hasn't change there.

**Comment by weibullguy** [comment]
Sorry @finswimmer, I didn't catch that this is a module docstring (at the top of the file); I assumed it was docstring in a function/method.  It should be fixed in v1.7.3-rc4.

**Comment by finswimmer** [comment]
`v1.7.3-rc4` fixed this :partying_face: Thanks a lot :pray:
